UT4MasterServer/Controllers/UnrealTournamentStatsController.cs

```diff
@@ -96,4 +96,11 @@ public async Task<IActionResult> GetAllTimeAccountStatistics(string id)
 		await statisticsService.CreateAccountStatisticsAsync(accountId, ownerType, statisticBase);
 		return Ok();
 	}
+
+	[HttpDelete]
+	public async Task<IActionResult> DeleteOldStatistics(int days = 30, bool skipFlagged = true)
+	{
+		var result = await statisticsService.DeleteOldStatisticsAsync(days, skipFlagged);
+		return Ok(result);
+	}
```

Comment:
There have to be some extra authorization checks somewhere. Please wait until #102 merges and then make sure account has sufficient permissions to run this action.

Reply:
Agreed to remove this endpoint until needed.